src/stories/Button.stories.mdx

```diff
@@ -818,62 +818,64 @@ Alternative pattern for non-standard icon sizes
 <a-button size="auto" label="Settings"><your-icon /></a-button>
 ```
 
-<Canvas>
-  <Story
-    name="icon auto"
-    args={{
-      label: 'Close',
-      size: 'auto',
-      default: '<a-icon name="Close" size="sm" />'
-    }}>
-    {Template.bind({})}
-  </Story>
-  <Story
-    name="icon auto subtle"
-    args={{
-      variant: 'subtle',
-      label: 'Close',
-      size: 'auto',
-      default: '<a-icon name="Close" size="sm" />'
-    }}>
-    {Template.bind({})}
-  </Story>
-  <Story
-    name="icon auto primary"
-    args={{
-      variant: 'primary',
-      label: 'Close',
-      size: 'auto',
-      default: '<a-icon name="Close" size="sm" />'
-    }}>
-    {Template.bind({})}
-  </Story>
-  <Story
-    name="icon auto disabled"
-    args={{
-      disabled: true,
-      label: 'Close',
-      size: 'auto',
-      default: '<a-icon name="Close" size="sm" />'
-    }}>
-    {Template.bind({})}
-  </Story>
-  <Story
-    name="icon auto pressed"
-    args={{
-      pressed: true,
-      label: 'Close',
-      size: 'auto',
-      default: '<a-icon name="Close" size="sm" />'
-    }}>
-    {Template.bind({})}
-  </Story>
-</Canvas>
+<Story
+  name="icon auto"
+  args={{
+    label: 'Close',
+    size: 'auto',
+    default: '<a-icon name="Close" size="sm" />'
+  }}>
+  {Template.bind({})}
+</Story>
+<Story
+  name="icon auto subtle"
+  args={{
+    variant: 'subtle',
+    label: 'Close',
+    size: 'auto',
+    default: '<a-icon name="Close" size="sm" />'
+  }}>
+  {Template.bind({})}
+</Story>
+<Story
+  name="icon auto primary"
+  args={{
+    variant: 'primary',
+    label: 'Close',
+    size: 'auto',
+    default: '<a-icon name="Close" size="sm" />'
+  }}>
+  {Template.bind({})}
+</Story>
+<Story
+  name="icon auto disabled"
+  args={{
+    disabled: true,
+    label: 'Close',
+    size: 'auto',
+    default: '<a-icon name="Close" size="sm" />'
+  }}>
+  {Template.bind({})}
+</Story>
+<Story
+  name="icon auto pressed"
+  args={{
+    pressed: true,
+    label: 'Close',
+    size: 'auto',
+    default: '<a-icon name="Close" size="sm" />'
+  }}>
+  {Template.bind({})}
+</Story>
 
 ## Using the label prop
 
 ```html
 <a-button label="Button"></a-button>
+<a-button label="Primary Menu">
+  Primary
```

Comment:
was this intentional?

Reply:
yes, I wanted to give a better example in the absence of the canvas's source code

Reply:
I meant having a "Primary Menu" label and a "Primary" content

Reply:
kind of? might not have been the best example, but basically it's about the fact that if you have a slot content, and the label prop, the label becomes the value of aria-label attribute